strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/test/java/org/carlspring/strongbox/testing/artifact/PythonWheelTestArtifact.java

```diff
@@ -0,0 +1,49 @@
+package org.carlspring.strongbox.testing.artifact;
+
+import org.carlspring.strongbox.artifact.generator.PythonWheelArtifactGenerator;
+import org.carlspring.strongbox.storage.Storage;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import org.apache.maven.model.Repository;
+import org.springframework.core.annotation.AliasFor;
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * @author Yuri Zaytsev
+ *
+ */
+@Documented
+@Retention(RUNTIME)
+@Target({ PARAMETER, ANNOTATION_TYPE})
+@TestArtifact(generator = PythonWheelArtifactGenerator.class, strategy = PythonWheelArtifactGeneratorStrategy.class)
+public @interface PythonWheelTestArtifact
```

Comment:
there also should be a `resource` attribute here (see `TestArtifact.resource()`).